presto-main/src/main/java/io/prestosql/type/UuidOperators.java

```diff
@@ -46,6 +46,7 @@
 import static io.prestosql.spi.function.OperatorType.XX_HASH_64;
 import static io.prestosql.type.UuidType.UUID;
 import static java.util.UUID.randomUUID;
+import static java.util.UUID.nameUUIDFromBytes;
```

Comment:
This should go to line 48.

Reply:
Okay, you mean like `import static java.util.UUID.*` ? Is importing all packages good practice?

Reply:
Okay, got it, it should be alphabetic.